dashboard/test/models/pd/application/teacher_application_test.rb

```diff
@@ -797,9 +797,13 @@ class TeacherApplicationTest < ActiveSupport::TestCase
         # Updates the application with principal's responses and run auto_score! again
         application.on_successful_principal_approval_create(principal_approval)
 
-        assert_equal test_case[:meet_requirement],
```

Comment:
This was showing a warning saying it should be `refute` ... not sure this is the best way to refactor, but it does eliminate the warning

Reply:
Ah, that's great––will do that instead

Reply:
Oh wait, this isn't from rubocop. This is a `Minitest 6` deprecation warning:

`DEPRECATED: Use assert_nil if expecting nil from test/models/pd/application/teacher_application_test.rb:800. This will fail in Minitest 6.`

I'll add a comment for this, unless you have a different suggestion

Reply:
ahh good point. in that case a comment on your current implementation sounds good to me.